File: Python/Additonal Questions/convert.py
def checkFactor(num):
    for i in range(2,num):
        if num % i == 0:
            return i

a,b = map(int,input().split())
s = a
flag = False
res = 0

i = 1
while(i <= a):
    res = checkFactor(s)
    s += res

    if s == b:
        flag = True
        break
    i += 1

if flag:
    print("Yes")
else:
    print("No")

File: C++/convert.cpp
#include <bits/stdc++.h>
using namespace std;

int factor(int num) {
    for(int i = 2; i<num;i++) {
        if(num%i == 0) {
            return i;
        }
    }
}

int main() {
    int a,b;
    cin >> a >> b;
    int s = a;
    bool flag = false;
    int res = 0;

    int i = 1;
    while(i <= a){
        res = factor(s);
        s += res;

        if(s == b){
            flag = true;
            break;
        }
        i += 1;
    }

    if(flag) {
        cout << "Yes";
    } else {
        cout << "No";
    }

    return 0;
}